zeebe/Dockerfile

```
@@ -161,7 +161,7 @@ RUN groupadd --gid 1001 camunda && \
     chown -R 1001:0 ${ZB_HOME} && \
     chmod -R 0775 ${ZB_HOME}
 
-COPY --link --chown=1001:0 docker/utils/startup.sh /usr/local/bin/startup.sh
+COPY --link --chown=1001:0 zeebe/docker/utils/startup.sh /usr/local/bin/startup.sh
```

```suggestion
COPY --link --chown=1001:0 docker/utils/startup.sh /usr/local/bin/startup.sh
```
So this assumes the build is run with a build context pointing to the repo root and is thus not specifying paths relative to the Dockerfile (which I would assume)?

Wdyt about setting `context: zeebe` on the `build-push-action` instead and keep paths here relative to the dockerfile location?